contracts/hts-precompile/examples/token-create/TokenCreateCustom.sol

```diff
@@ -23,7 +25,7 @@ contract TokenCreateCustomContract is HederaTokenService, ExpiryHelper, KeyHelpe
         bytes memory key
     ) public payable {
         IHederaTokenService.TokenKey[] memory keys = new IHederaTokenService.TokenKey[](5);
-        keys[0] = getSingleKey(KeyType.PAUSE, KeyValueType.SECP256K1, key);
+        keys[0] = getSingleKey(KeyType.ADMIN, KeyType.PAUSE, KeyValueType.SECP256K1, key);
```

Comment:
this looks broken, should be 3 param values.
Not sure what you're trying to change

Reply:
There are three `getSingleKey` functions within the KeyHelper.sol here is the one with 4 params https://github.com/hashgraph/hedera-smart-contracts/blob/main/contracts/hts-precompile/KeyHelper.sol#L96. 

I was trying to set up both ADMIN and PAUSE type at keys[0] as the minting and grantingKYC test cases didn't work without ADMIN type set. I got the reference from this method https://github.com/hashgraph/hedera-smart-contracts/blob/main/contracts/hts-precompile/examples/token-create/TokenCreateContract.sol#L28

Before when the ADMIN type is not set and the test suites still all worked as there is only one test utilizing the `createFungibleTokenPublic` https://github.com/hashgraph/hedera-smart-contracts/blob/main/test/hts-precompile/token-create/tokenCreateContract.js#L369, and this test case didn't require updating token keys to mint or grant token KYC.

I just push a new commit which remove the .only and all CI tests pass now

Reply:
I see
Q: Would it be cleaner to leave the admin key as such and add a `keys[4] = getSingleKey(KeyType.PAUSE, KeyValueType.SECP256K1, key);`?

Reply:
Did you mean replace `keys[4] = getSingleKey(KeyType.WIPE, KeyValueType.SECP256K1, key);` with `keys[4] = getSingleKey(KeyType.PAUSE, KeyValueType.SECP256K1, key);` or you meant some thing like this 
```
        IHederaTokenService.TokenKey[] memory keys = new IHederaTokenService.TokenKey[](6);
        keys[0] = getSingleKey(KeyType.ADMIN, KeyValueType.SECP256K1, key);
        keys[1] = getSingleKey(KeyType.KYC, KeyValueType.SECP256K1, key);
        keys[2] = getSingleKey(KeyType.FREEZE, KeyValueType.SECP256K1, key);
        keys[3] = getSingleKey(KeyType.SUPPLY, KeyValueType.SECP256K1, key);
        keys[4] = getSingleKey(KeyType.WIPE, KeyValueType.SECP256K1, key);
        keys[5] = getSingleKey(KeyType.PAUSE, KeyValueType.SECP256K1, key);
```

Replacing doesn't work for minting and granting either but adding one more key to the `keys` array works.

Reply:
I meant adding the key to the array

Reply:
Yes! It works like that too! Will push in the update soon!